app/assets/stylesheets/admin/disabled.scss

```diff
@@ -16,4 +16,8 @@ input[type='button'], input[type='submit'] {
   .select2-choice > .select2-chosen {
     color: #a1a1a1;
   }
+
+  .select2-search-choice {
+    background-color: #c3c3c3 !important;
+  }
```

Comment:
I took the liberty of disabling the selector box if the ordering method is not selected (if "By producer" is selected, the "Category" selector is disabled). Then I added this class because the cards lables were not readable for the disabled box (white text on light gray background). Please let me know if I should declare this in a different way, or if the selector box should be always enabled for both methods

Reply:
I think it's fine, I am just wondering if there is a better css way to do that without  !important.

Reply:
I maybe missed something, but I cannot find any `div` with class that is `select2-search-choice`. 
I just find this CSS rule that is used to gray the disabled field:
```css
.select2-container-multi.select2-container-disabled .select2-choices {
  background-color: #f4f4f4;
}
```

available in the `select2-rails` gem.

Could you please explain me which part of the part you want to disable with this rule?
Thanks a lot! 🙏